code-of-conduct.md

```diff
@@ -14,45 +14,35 @@ race, religion, or sexual identity and orientation.
 We are committed to providing a friendly, safe and welcoming environment for
 all.
 
-On the Adafruit Discord server, please avoid using nicknames or avatar images
-that might detract from a friendly, safe and welcoming environment. This
-includes but is not limited to use of overtly sexual, vulgar, or otherwise
-offensive nicknames or avatar images.
-
-Please avoid excessive helping. Excessive helping is defined as providing help
-that was not asked for. If someone asks a question, be sure that you're
-providing the help that was requested and not providing unnecessary information
-beyond the scope of the question. Allow the person asking the question to drive
-the conversation. Ask the person to clarify if you're unsure what they are
-asking, instead of providing an answer that covers the question *you think*
-they're trying to ask. If you feel the need to "take the keyboard away" from
-the person requesting assistance, reconsider how you're trying to answer. If
-you're doing all the talking, you may be excessively helping.
-
 Examples of behavior that contributes to creating a positive environment
 include:
 
 * Be kind and courteous to others
-* Behaving professionally
 * Using welcoming and inclusive language
 * Being respectful of differing viewpoints and experiences
+* Collaborating with other community members
 * Gracefully accepting constructive criticism
 * Focusing on what is best for the community
 * Showing empathy towards other community members
 
 Examples of unacceptable behavior by participants include:
 
-* The use of sexualized language or imagery and unwelcome sexual attention or
-  advances
-* The use of inappropriate images in a user's avatar or inappropriate nicknames
+* The use of sexualized language or imagery and sexual attention or advances
+* The use of inappropriate images, including in a user's avatar
+* The use of inappropriate language, including in a user's nickname
 * Any spamming, flaming, baiting or other attention-stealing behavior
-* Excessive helping
+* Excessive or unwelcome helping; answering outside the scope of the question
+  asked
 * Trolling, insulting/derogatory comments, and personal or political attacks
 * Public or private harassment
 * Publishing others' private information, such as a physical or electronic
   address, without explicit permission
-* Other conduct which could reasonably be considered inappropriate in a
-  professional setting
+* Other conduct which could reasonably be considered inappropriate
+
+We value many things beyond technical expertise, including collaboration and
+supporting others within our community. Providing a positive experience for
+other community members can have a much more significant impact than simply
+providing the correct answer.
```

Comment:
Yes! Love this section! "answering outside the scope of the question asked" is perfectly worded

Reply:
Thank you!

Reply:
I have to disagree with @tannewt. While I assume to understand the intend of the sentence "Excessive or unwelcome helping; answering outside the scope of the question asked", I have to disagree with it's implications; sometimes it seems sensible to me to give the mostly unwanted piece of advice, that "you should not be doing this".

Reply:
@NonsenseInc Thank you so much for the feedback!

There are absolutely situations where that fits! Giving the advice of "You should not be doing this" is not considered answering outside the scope of the question asked. If applicable, it's providing good advice within the scope of the question asked. What we're trying to avoid here is the person answering overwhelming the person asking with information outside what they asked, or taking over the conversation to the point where the person asking is no longer really driving it. Initially we had the phrase "taking the keyboard away from" which is a reference to being in person, trying to assist someone with an issue on their own computer, and instead of helping them through the issue, you take the keyboard away from them and do the fix yourself. We didn't want to have an entire paragraph specific to explaining what we mean by excessive helping because we thought it made it seem more important than the other examples of inappropriate behaviors. The concept needs to be included because unfortunately there are people who engage in this behavior, and we want to make sure that it's clear that it is inappropriate.

I agree entirely with your sentiment, but I don't think this wording applies to your example.

Reply:
@kattni Thank you very much for your explaination! In conclusion I think I misunderstood the rule and now agree with it and it's wording.